DedicatedServer/Scripts/Modes/Trackmania/FlagRush.Script.txt

```diff
@@ -799,12 +799,13 @@ Void SpawnPlayers() {
 		if (Player.SpawnStatus == CSmPlayer::ESpawnStatus::NotSpawned) {
```

Comment:
Since you now changed the variable name for spawn date and I originally wanted to do a fix with it in another ticket, could you add a spawn date to the if clause above? 
```
if (Player.Armor <= 0) {
	FlagRush_UnspawnPlayer(Player);
	// Spawn Date here
}
```